src/JuliusSweetland.OptiKey.Chat/App.xaml.cs

```diff
@@ -199,6 +199,12 @@ private void App_OnStartup(object sender, StartupEventArgs e)
                 {
                     mainWindowManipulationService.SizeAndPositionInitialised += sizeAndPositionInitialised;
                 }
+
+                Current.Exit += (o, args) =>
+                {
+                    mainWindowManipulationService.PersistSizeAndPosition();
```

Comment:
@kmcnaught Do you know if there is the potential for this to incorrectly persist a temporary window state, e.g. in a dynamic keyboard which has temporarily resized optikey?